frontend/src/actions/parcelsActions.ts

```diff
@@ -20,6 +20,7 @@ export interface IProperty {
   description?: string;
   projectNumber?: string;
   projectStatus?: string;
+  isSensitive: boolean | '';
```

Comment:
just cleaning up some typings.

Reply:
Can this be changed to a nullable boolean?

Reply:
it isn't a nullable boolean, because formik won't accept null/undefined as an initial value.